move/walrus_site/sources/site.move

```diff
@@ -52,11 +59,35 @@ module walrus_site::site {
         route_list: vec_map::VecMap<String, String>,
     }
 
+    /// One-Time-Witness for the module.
+    public struct SITE has drop {}
+
+    fun init(otw: SITE, ctx: &mut TxContext) {
+        let publisher = package::claim(otw, ctx);
+        set_site_display(&publisher, ctx);
+        transfer::public_transfer(publisher, ctx.sender());
+    }
+
     /// Creates a new site.
-    public fun new_site(name: String, ctx: &mut TxContext): Site {
+    public fun new_site(
+        name: String,
+        link: Option<String>,
```

Comment:
I think all of them should be moved into a single `Metadata` field which can be edited / set / updated in a single call or copied over. Too many arguments is not a good strategy for usability.

Reply:
Add a separate module with a type `public struct Metadata { fields } has copy, drop, store` and accept it as an argument here.